### PR TITLE
Adds Modified Readme file

### PR DESCRIPTION
**Pull request by Molly872**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
 [![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/cSGmFTKd)
-# INF 6420 - Intro to Web Development
+# INF 6420 - Molly's Project Website
+
+I thought it would be fund to create a faux rendition of a virtual book club.  Forgive my bad graphics skills, things will be more even on creation. These are very simple designs.  
+
+## Wireframes
+
+### About Me Page
+![](WireFrames\About-Me-Page-Wireframe.png)
```
Comment (axamei) [inline README.md]: Please change `\` to `/`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+#### Header - Logo or Word Art
+Navigation Menu - Will link to other topic pages
+Main Content - Photo and about me section
+Footer - Book Club Title and Date
+
+### Landing Page/Book List
+
+![](WireFrames\Book-List-(Landing-Page)-Wireframe.png)
```
Comment (axamei) [inline README.md]: Please change `\` to `/`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+
+#### Header - Logo or Word Art
+Navigation Menu - Will link to other topic pages
+Main Content - Photos of books as links to individual pages and descriptions/reviews of each book
+Footer - Book Club Title and Date
+
+### Individual Book Pages
+
+![](WireFrames\Book-Review-Ind-Book-Wireframe.png)
```
Comment (axamei) [inline README.md]: Please change `\` to `/`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,41 @@
...
+
+#### Header - Logo or Word Art
+Navigation Menu - Will link to other topic pages
+Main Content - Book Photo, Title, Description and Attempted Comment Section
+Footer - Book Club Title and Date
+
+### Make A Recommendation
+
+![](WireFrames\Make-A-Recommendation-Page-Wireframe.png)
```
Comment (axamei) [inline README.md]: Please change `\` to `/`

**Review by axamei — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Molly872) [reply README.md]: Updated.  Let me know if you don't see it

File: README.md (hunk 2 above)
Comment (Molly872) [reply README.md]: Updated. Thanks!

File: README.md (hunk 3 above)
Comment (Molly872) [reply README.md]: Updated, thanks!

File: README.md (hunk 4 above)
Comment (Molly872) [reply README.md]: updated, thanks!

**Comment by axamei** [comment]
@Molly872 I don't see the change. Can you make sure your changes are committed and then pushed? Book an appointment if you need clarity.

**Review by axamei — APPROVED** [review]

**Pull request merged**